pkg/apis/migration/migration.go

```diff
@@ -47,12 +47,19 @@ import (
 	k8s "sigs.k8s.io/vsphere-csi-driver/pkg/kubernetes"
 )
 
+// VolumeSpec contains VolumePath and StoragePolicyID
+// using which Volume can be looked up using VolumeMigrationService
+type VolumeSpec struct {
```

Comment:
How about renaming VolumeSpec ==> MigrationVolumeSpec or VolumeMigrationSpec ?
Would it be too specific ?

Reply:
`VolumeSpec` type is part of `migration` package.  So I think it is ok to not prefix or postfix with keyword `migration`.
what is your opinion? Is it creating confusion, with volume create spec, query volume spec ...

Reply:
That's right! Caller will always migration.VolumeSpec. This makes sense.
We can keep it as VolumeSpec.